scripts/visualize/visualize.matthew_map.R

```diff
@@ -25,9 +27,16 @@ visualize.matthew_map <- function(viz){
   
   for (i in 1:length(counties)){
     xml_attr(p[[i]], 'id') <- paste0('FIP-', as.character(counties$FIPS[i]))
+    xml_attr(p[[i]], 'class') <- 'county-polygon'
     xml_attr(p[[i]], 'style') <- NULL
     xml_attr(p[[i]], 'clip-path') <- NULL
   }
   
+  g.rivers <- xml_add_child(svg, 'g', id='rivers','class'='river-polyline')
```

Comment:
we don't ID rivers individually, so I grouped all of them into a single group id as `rivers` and class `river-polygon`